src/auth/digest/Config.cc

```diff
@@ -22,6 +22,7 @@
 #include "auth/State.h"
 #include "auth/toUtf.h"
 #include "base/LookupTable.h"
+#include "base/Random.h"
```

Comment:
This and probably all other files that are using the new seeding API now `#include` both base/Random.h and `<random>` (not visible in the diff). I think that is the best approach because the fact that base/Random.h also includes `<random>` is not something the seeding API is meant to guarantee. It was just easier to implement those functions using `<random>` types. However, if others insist, we will remove `<random>` from this and similar files.

Reply:
I do object. Please minimize the dependency include tree wherever possible.

Reply:
> I do object. Please minimize the dependency include tree wherever possible.

Will do.

P.S. Please note that it is impractical to "minimize the dependency include tree wherever possible" -- such minimization is always "possible", but the amount of resources required to do that usually outweighs the benefits (by orders of magnitude!), and the changes required to achieve that minimum would often go out of the PR scope. "Minimum possible" cannot be our standard (until we have the necessary tools to achieve it without manual labor).

Reply:
Done (30e4b89)